src/App.js

```diff
@@ -76,10 +78,8 @@ function App() {
       if (!response.ok) {
         throw new Error(`HTTP error! Status: ${response.status}`);
       }
-      const newList = todoList.filter(item =>
-        item.id !== id
-      )
-      setTodoList(newList)
+
+      setTodoList(todoList.filter(item => item.id != id))
```

Comment:
Requires change:
1. You should be using strictly equal `!==` instead of loosely equals `!=` here
2. Instead of filtering the `todoList`, you can use a callback function to filter the `prevTodoList`:
```diff
- setTodoList(todoList.filter((todo) => todo.id != id));
+ setTodoList(prevTodoList => prevTodoList.filter(todo => todo.id !== id));
```

Reply:
Not sure how I missed the strictly equals, definitely need to change that. Callback function too.